### PR TITLE
Replace various config files with a single config

### PR DESCRIPTION
**Pull request by bsweger**
This is the re-factor for [DB-296](https://federal-spending-transparency.atlassian.net/browse/DB-296).

* Remove app init scripts that prompt for config settings
* Code now gets config settings via a dataactcore.config import
* Interfaces now get db names from config

There will be a subsequent PR that includes the corresponding doc changes.

### CODE REVIEW

**Comment by bsweger** [comment]
@jworcestBAH and @nmonga91 this is ready for review--thanks!

File: dataactbroker/app.py [hunk 1]
```diff
@@ -59,20 +57,21 @@ def createApp():
         def root():
             return "Broker is running"
 
-        localFiles = "".join([config["local_folder"], "/<path:filename>"])
+        localFiles = os.path.join(broker_file_path, "/<path:filename>")
```
Comment (nmonga91) [inline dataactbroker/app.py]: If any argument in os.path.join() has a leading "/" (making it an absolute path), the other arguments are then ignored. It doesn't look like that is the desired functionality here, so I would recommend removing the leading slash in the 2nd argument.

Comment (bsweger) [inline dataactbroker/app.py]: Great catch! Fixed...thank you.

File: dataactbroker/app.py [hunk 2]
```diff
@@ -59,20 +58,22 @@ def createApp():
         def root():
             return "Broker is running"
 
-        localFiles = "".join([config["local_folder"], "/<path:filename>"])
```
Comment (jworcestBAH) [inline dataactbroker/app.py]: When running with use_aws set to true, "local_folder" gets set to None which breaks this line.  I've enclosed this and the dependent route in an "if local" block.

**Pull request merged**